<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mychat</class>
 <widget class="QDialog" name="mychat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textInput">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="MsgBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="textFont">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="textBold">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="textItalic">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="textUnderline">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="textSize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="textColor">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>✨</string>
   </property>
  </widget>
  <widget class="QToolButton" name="textClear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
